lib/valicon/conversions.ex

```diff
@@ -140,4 +140,13 @@ defmodule Valicon.Conversions do
       {value, rest} -> Map.put(rest, to, value)
     end
   end
+
+  @spec atomize_values(map(), [atom()]) :: map()
+  def atomize_values(map, possible_values) do
```

Comment:
Why don't you want to specify the key? You most likely don't need to convert all the keys in the map

Reply:
just update the implementation a little bit because `Map.update/3` does not exist, only `Map.update!/3`